src/downloader/apkmirror.py

```diff
@@ -76,6 +77,8 @@ def get_download_page(self: Self, main_page: str) -> str:
     @staticmethod
     def _extracted_search_div(url: str, search_class: str) -> Tag:
         """Extract search div."""
+        logger.info("Pausing for 15 seconds...")
+        time.sleep(15)
```

Comment:
Not everyone build 6-7 apps. I think it will be bad experience for those who are building only 1-2 apps. 
What we can do it instead of hard coding 15sec in the code itself. We can expose it as an option just like other options.

Reply:
Wouldn't it clutter the .env? We already have a lot.

I guess, I'll just use it to myself till a better workaround is implemented, if ever.

Reply:
I don't think it will clutter. It is going not going to be a mandatory field. You can also choose to keep this to your fork. You can close the PR in that case.